AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts

```diff
@@ -157,4 +157,8 @@ export class VhoWorkHoursTableComponent implements CanDeactiveComponent {
     registerUnsavedChanges() {
         this.videoHearingsService.setVhoNonAvailabiltiesHaveChanged(true);
     }
+
+    get checkVhoHasWorkHours(): boolean {
```

Comment:
A recommended naming style for methods that return a bool is to prefix with with 'has' or 'is' to better reflect the behaviour of the method as a question. For example, `hasVhoGotWorkingHours` (assuming I got the context of the method right there)